### ISSUE TITLE
/api/message/set

### ISSUE THREAD

**Issue by ryunalore** [ryunalore/SLM#7]
- Provide function that requires parmeters
- When executed it changes the "current state"
   - example no.1 - state has been change:
       ``` 
           1. execute /api/message/set?m=Service+checks:+No+power+until+5:00+pm
           2. result: "ok"
           3. execute /api/message
           4. "Service checks: No power until 5:00 pm"
       ```
- Realise as POST Request
- Return HTTP Status Code 200 (successful)

**Comment by ryunalore** [comment]
@ic21b006 @kaplanilhan Code is now implemented  (see commit [6eede86](https://github.com/ryunalore/SLM/commit/6eede86feab265a72314e70a41a2c54fc70bda13) )


![Image](https://user-images.githubusercontent.com/96705308/204165569-0a64cb1d-0cf8-47e6-a190-3467585cd88e.png)


![Image](https://user-images.githubusercontent.com/96705308/204165580-9e5f09cd-cabf-447b-bb4a-54399ccfcf36.png)

**Comment by ryunalore** [comment]
changed to @GetMapping
